onepager.py

```diff
@@ -98,6 +96,15 @@
 standard_bool is not None                       # = True
 type(standard_bool) == bool                     # = True
 
+x = ['a', 'b', 'c']
+y = x                                           # x and y reference the same object
+z = ['a', 'b', 'c']                             # x and z reference different objects with identical data
+x is y                                          # = True, comparison by reference
+x is z                                          # = False, comparison by reference
+x == z                                          # = True, comparison by value
+x = ['c', 'c', 'c']                             # = Reassign x to a new object
+x is y                                          # = False, y still points to original object
+
```

Comment:
Why classes specifically? I wasn't aware Python behaved that way.... Is it only with simple data types? Is it some sort of space saving mechanism?

Reply:
Python does indeed do that to save on memory since a language like C/C++, etc., have values stored on stack for 1, etc., it doesn't need it, but since Python keeps more on the heap, it has to do memory cutting things like keeping one copy of some value types like integers and why classes was because usually you're comparing instances of classes being the same.

Maybe move this out and we'll do a final ticket to add it.